.github/workflows/build-and-release-single-package.yml

```diff
@@ -165,3 +163,10 @@ jobs:
         shell: bash
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Ensure Taipy release is marked as latest
+        run: |
+           gh release edit ${{needs.fetch-versions.outputs.taipy_VERSION}} --latest
```

Comment:
I don't understand, where is the `taipy_VERSION` defined?

Reply:
Oh, it comes from the fech version step. A output variable was missing.